interface/reports/external_data.php

```diff
@@ -138,24 +59,25 @@
             $menuPatient = new PatientMenuRole();
             $menuPatient->displayHorizNavBarMenu();
             ?>
-            <div class="clear"></div>
             <div class="row">
```

Comment:
Add some spacing between the navbar and the nav pills to prevent it from looking like they're colliding
![image](https://user-images.githubusercontent.com/46497068/91072089-01592380-e607-11ea-850a-724a886799fa.png)